### PR TITLE
Feature/58445 freeze a pupil

### PR DESCRIPTION
**Pull request by jon-shipley**
SM Feature: Pupil Thaw

Change a pupil from a frozen (read-only) state back to an editable state.

### CODE REVIEW

File: admin/views/service-manager/pupil-summary.ejs [hunk 1]
```diff
@@ -23,10 +23,17 @@
               <li>
               <% if (pupil.isAnnulled) { %>
                 <a class="govuk-link" href="/service-manager/annul-pupil-undo/<%=pupil.urlSlug %>" >Undo Annulment</a>
-                <% } else { %>
+                <% } else if (!pupil.isFrozen) { %>
```
Comment (GuyHarwood) [inline admin/views/service-manager/pupil-summary.ejs:26]: Hiding the annulment when frozen could be confusing for the user.  I would have thought it be perfectly acceptable to go straight to annulment from frozen, given that this is one of the two outcomes from the freeze period.  Should we ask for clarification?

**Review by GuyHarwood — COMMENTED** [review]
Looks good, just one item to address

File: admin/views/service-manager/pupil-summary.ejs (hunk 1 above)
Comment (jon-shipley) [reply admin/views/service-manager/pupil-summary.ejs:26]: That's a good point.  I'll change it.

**Review by GuyHarwood — APPROVED** [review]
LGTM

**Comment by activemq** [comment]
Closing, covered in #2504

**Pull request closed without merging**